### PR TITLE
Style guide

### LINKED ISSUE #62: Style guide for prototype kit instructions
As lots of us work on the same set of instructions, some form of MVP style guide would help me and possibly others. 

## Terms so far I've noticed that I'd like documented

* NHS.UK prototype kit - the things that is has to have the NHS.UK together (not just NHS.UK) and how we talk about it after initially mentioning it ('the kit' seems to be what has been decided).
* GOV.UK Prototype Kit and GOV.UK Design System - admittedly not even documented in their own stuff maybe?
* NHS.UK frontend - this is probably defined somewhere else - maybe just linking to it
* Visual Studio Code - after installing things myself I think that it's not helpful to call it VSCode as that's not what it's called and doesn't even show up in computer searches.
* other things like 'command line' and 'terminal' etc - is it 'the terminal' and so on.

## Where this list could live

* in github e.g. part of README - could be a good starting point as it's only needed for the contributors
* in the actual documentation, for example part of contributing - could be a good place to live as there probably aren't too many phrases
* part of the NHS.UK service manual - maybe end game? and some of our stuff may be too niche

## Other thoughts
This is perhaps something that the GOV.UK Design System team could lead on or we start and hand over to them? This feels like a bit of a mix of style guide but also technical writing as this is very much technically focused.

### PR DESCRIPTION
**Pull request by vickytnz**
starting point for style guide. 

Things to check:
 * all the points really
 * anything re formatting. I thought this would be a definition list but the health guide A-Z is headings? 
 
Fixes #62 
 
 > ### Style guide
 >
 >We normally use the [NHS content guide](https://service-manual.nhs.uk/content) for terms. But there are some terms that are only used in the guidance for the kit:
 >
 > ### command line
 >
 >Lower case. Only use for Windows instructions. Nortmally it will be ‘the command line’.
 > 
 > #### GOV.UK Prototype Kit, GOV.UK Design System and other GOV.UK products
 >
 > Title case.
 >
 > #### NHS, NHS.UK, NHS England
 >
 > Use ‘NHS.UK’ for products that are for delivery teams and not the public (find out more on the [NHS health writing A-Z](https://service-manual.nhs.uk/content/a-to-z-of-nhs-health-writing#N))This includes the NHS.UK prototype kit, NHS.UK design system, NHS.UK service manual or NHS.UK website.
 >
 > #### NHS England laptops
 > Use ‘NHS England Windows laptops’ instructions for people using corporate laptops.
 >
 > #### NHS.UK prototype kit
 >
 > ‘NHS.UK’ is in caps and ‘prototype kit’ in lower case (except where the NHS.UK is shown as a logo when it is then presented as ‘NHS Prototype kit’). Write out in full the first time it is used on a page, then call ’the kit’. Do not use ‘prototype kit’ if you can.
 >
 > #### terminal
 > 
 > Lower case. Only use for Mac and Linux instructions. Usually it will be ‘the terminal’.

Things I'd like to add:
* anything about file paths - for example do we always need to spell them out as from the top or is it OK to be a bit more loose about them?
* more about the terminal and command prompt stuff, maybe I don't have enough yet to complete that

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -13,6 +12,33 @@ Start the code by running `npm run watch`.
 
 If you want to contribute to the NHS.UK prototype kit guidance, then read our [contributing guidelines](CONTRIBUTING.md).
 
+### Style guide
+
+We normally use the [NHS content guide](https://service-manual.nhs.uk/content) for terms. But there are some terms that are only used in the guidance for the kit:
+
+### command line
+
+Lower case. Only use for Windows instructions. Nortmally it will be ‘the command line’.
```
Comment (edwardhorsford) [inline README.md:21]:
```suggestion
Lower case. Only use for Windows instructions. Normally it will be ‘the command line’.
```

**Review by edwardhorsford — DISMISSED** [review]

**Review by vickytnz — COMMENTED** [review]
Changed

**Comment by vickytnz** [comment]
From Sara - looking to change from NHS.UK to NHS. 
This means opening this ticket up again @sarawilcox

**Comment by sarawilcox** [comment]
Hi @vickytnz , for GOV.UK and NHS we lower case "prototype kit": GOV.UK prototype kit. No need for title case. Ditto GOV.UK design system. (There are some inconsistencies in the service manual but we generally use sentence case for service names.)

We avoid using "NHS.UK" as it confuses people. Some people think we're referring to the NHS.UK programme. Other people think we're talking about the wide nhs.uk domain. So we tend to write: NHS prototype kit, NHS design system, NHS digital service manual and "the NHS website (at www.nhs.uk)". Again there are some inconsistencies, particularly with the frontend and our GitHub repos. Part of the reason we prefer NHS to NHS.UK is that it makes it clear that the design system, prototype kit etc are for the whole NHS, regardless of programme or domain.

I have been using "prototype kit" but could switch to "kit" (after first spelling NHS prototype kit out in full). I was wondering why you say not to use "prototype kit"?

**Pull request merged**